API.md

```diff
@@ -1,4 +1,111 @@
-# Authmosphere API documentation
+# Authmosphere API Documentation
+
+## OAuth Tooling
+
+This tooling provides helper functions to request and validate tokens based on the OAuth 2.0 [RFC 6749](https://tools.ietf.org/html/rfc6749) specification.
+
+### getAccessToken
+
+Requests a token based on the given configuration (which specifies the grant type and corresponding parameters). See the [`OAuthConfig` documentation](#tbd) for details.
+
```

Comment:
#tbd will be substituted, when document exists?

Reply:
Exactly